esy/Package.ml

```diff
@@ -53,6 +53,34 @@ module CommandList = struct
 
 end
 
+(**
+ * Scripts with keys as specified in "scripts".
+ *)
+module Scripts = struct
+
+  type script = {
+    key : string;
+    command : string;
+  }
+  [@@deriving (show, eq, ord)]
+
+  type t =
+    script list
```

Comment:
Maybe we should use `script Map.Make(String)` to store scripts? This would fit better with the access pattern we are going to use for scripts — "lookup a single script and execute it".